src/components/common/program-section-templates/single-image-right/SingleImageRight.module.scss

```
@@ -51,6 +51,14 @@
     align-items: start;
     > div {
         flex: 1;
+        width: 100%;
+
+        textarea {
+            word-wrap: break-word;
+            word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)